build/android/bytecode/java/org/brave/bytecode/BraveSiteSettingsDelegateAdapter.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.brave.bytecode;
+
+import org.objectweb.asm.ClassVisitor;
+
+public class BraveSiteSettingsDelegateAdapter extends BraveClassVisitor {
+
+    static String sChromeSiteSettingsDelegateClassName = "org/chromium/chrome/browser/site_settings/ChromeSiteSettingsDelegate";
```

Comment:
Please add test to check that class still exist, constructor check covers it, but it is faster to find the issue if we check class existence first.